lib/rubocop/cop/rails/skips_model_validations.rb

```diff
@@ -25,10 +25,22 @@ module Rails
       class SkipsModelValidations < Cop
         MSG = 'Avoid using `%s` because it skips validations.'.freeze
 
+        REQUIRE_ARGUMENT = %w(decrement!
```

Comment:
This name is a bit confusing. How about `METHODS_WITH_ARGUMENTS` or something like this?